### DISCUSSION TITLE
Migrate from pgvecto.rs to VectorChord

### DISCUSSION THREAD

**Discussion by ngosang** [immich-app/immich#14640] [General]
It's the successor  of pgvecto.rs, developed by the same team.
VectorChord 0.1.0 was released => https://github.com/tensorchord/VectorChord/releases/tag/0.1.0
There is a blog post with impressive benchmarks => https://blog.pgvecto.rs/vectorchord-store-400k-vectors-for-1-in-postgresql

Related => https://github.com/immich-app/immich/discussions/14280

**Comment by danieldietzler** [comment]
That is already our plan, yep :)

**Comment by martabal** [comment]
More context: https://github.com/tensorchord/VectorChord/issues/101
